lib/ingestor-api/runtime/src/schemas.py

```diff
@@ -52,7 +52,7 @@ def exists(cls, collection):
 
 class StacCollection(Collection):
     id: str
-    item_assets: Dict
+    item_assets: Optional[Dict]
```

Comment:
Something feels a bit off about this.  Either we're using the [Item Assets extension](https://github.com/stac-extensions/item-assets#collection-fields) or we're not.

Perhaps we should put this behind a feature flag?  Not really critical for this PR though.

Reply:
I thought we could make it optional since it's not required in the base STAC specification (didn't realize it's an extension in itself). In MAAP we have a couple collection files without this information. I agree feature flags sounds like a better way to handle this.

Reply:
Actually, I realized today in another deployment that the `Optional` flag actually means you're assigning `None` when not providing a value explicitly, which can create problems, so we shouldn't merge this as is.